Makefile

```diff
@@ -31,12 +34,22 @@ build:
 	$(CRI_BIN) build --build-arg BASE_IMAGE_TAG=$(CRI_BUILD_BASE_IMAGE_TAG) . -t $(REG)/$(ORG)/$(CHECKUP_IMAGE_NAME):$(CHECKUP_IMAGE_TAG)
 .PHONY: build
 
-push:
+push: push-trex
 	$(CRI_BIN) push $(REG)/$(ORG)/$(CHECKUP_IMAGE_NAME):$(CHECKUP_IMAGE_TAG)
 	$(CRI_BIN) tag $(REG)/$(ORG)/$(CHECKUP_IMAGE_NAME):$(CHECKUP_IMAGE_TAG) $(REG)/$(ORG)/$(CHECKUP_IMAGE_NAME):$(CHECKUP_GIT_TAG)
 	$(CRI_BIN) push $(REG)/$(ORG)/$(CHECKUP_IMAGE_NAME):$(CHECKUP_GIT_TAG)
 .PHONY: push
 
+build-trex:
+	$(CRI_BIN) build -f trex-config/Dockerfile -t $(REG)/$(ORG)/$(TREX_IMAGE_NAME):$(TREX_IMAGE_TAG) trex-config
```

Comment:
What is the meaning of `trex-config` at the end of this line?

Reply:
that's the folder where the Dockerfile and other scripts reside